src/Writer.jl

```diff
@@ -362,7 +362,7 @@ Arguments:
   • indent (optional number): if provided, pretty-print array and object
     substructures by indenting with the provided number of spaces
 """
-json(a) = sprint(print, a)
+json(a; indent=nothing) = ifelse(indent===nothing, sprint(print, a), sprint(print, a, indent))
```

Comment:
`ifelse` doesn't seem like a good idea, that evaluates both expressions and throws away one, perhaps you meant to use:

```jl
indent === nothing ? sprint(print, a) : sprint(print, a, indent)
```

Reply:
Thank you for your review. I fixed it.